Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -168,12 +168,9 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     moveFixIt: (_ misplacedTokens: [TokenSyntax]) -> FixItMessage,
     removeRedundantFixIt: (_ misplacedTokens: [TokenSyntax]) -> FixItMessage? = { _ in nil }
   ) {
-    guard let incorrectContainer = unexpected,
-      let misplacedTokens = incorrectContainer.onlyPresentTokens(satisfying: unexpectedTokenCondition)
-    else {
-      // If there are no unexpected nodes or the unexpected contain multiple tokens, don't emit a diagnostic.
-      return
-    }
+    guard let incorrectContainer = unexpected else { return }
+    let misplacedTokens = incorrectContainer.presentTokens(satisfying: unexpectedTokenCondition)
+    if misplacedTokens.isEmpty { return }
```

Comment:
Out of curiosity: What motivated this change?

Reply:
Ah, I see. Could you update the doc comment to match the new behavior?

Reply:
Updated. Please take a look